src/types/onyx/PurchaseList.ts

```diff
@@ -0,0 +1,188 @@
+/** Subscription type for a purchase */
+type Subscription = {
```

Comment:
This is the same as the `PrivateSubscription` type, minus the `errors` and `errorFields` - can we consolidate in some way so there is less duplication?